components/alert/test/alert-toast.vdiff.js

```diff
@@ -95,5 +105,68 @@ describe('alert-toast', () => {
 			await expect(document).to.be.golden();
 		});
 
+		describe('hover and focus', () => {
+
+			async function focusAlert(elem) {
+				const firstAlertButton = elem.querySelector('d2l-alert-toast').shadowRoot.querySelector('d2l-alert').shadowRoot.querySelector('d2l-button-icon');
+				await focusElem(firstAlertButton);
+			}
+
+			async function hoverAlert(elem) {
+				const firstAlert = elem.querySelectorAll('d2l-alert-toast')[1].shadowRoot.querySelector('d2l-alert');
+				await hoverElem(firstAlert);
+			}
+
+			async function hoverOtherElem(elem) {
+				const otherElem = elem.querySelector('div#other');
+				await hoverElem(otherElem);
+			}
+
+			let clock;
+
+			beforeEach(() => {
+				clock = sinon.useFakeTimers({ toFake: ['clearTimeout', 'setTimeout'] });
+			});
+			afterEach(() => {
+				clock.restore();
+			});
+
+			it('hover then wait', async() => {
+				const elem = await fixture(multipleAlertsAutoClose, { viewport: { width: 700, height: 400 } });
```

Comment:
Nit: I'd suggest pulling `{ width: 700, height: 400 }` into a `viewport` const to reuse across these.

```suggestion
				const elem = await fixture(multipleAlertsAutoClose, { viewport });
```